data/core/closetheloop.md

```diff
@@ -1,921 +0,0 @@
-    - slot{"language":"spanish"}
-    - action_store_bot_language
-    - slot{"can_use_spacy":true}
-    - utter_spacy_or_tensorflow
-    - utter_anything_else
-* enter_data{"language":"french"}
-    - slot{"language":"spanish"}
-    - action_store_bot_language
-    - slot{"can_use_spacy":true}
-    - utter_spacy_or_tensorflow
-    - utter_anything_else
-
-## Story from conversation with dominik on November 19th 2018
-* greet
-    - action_greet_user
-* how_to_get_started
-    - utter_getstarted
-    - utter_first_bot_with_rasa
-* deny
-    - action_set_onboarding
-    - slot{"onboarding":false}
-    - utter_ask_which_product
-* how_to_get_started{"product":"core"}
-    - slot{"product":"core"}
-    - utter_core_tutorial
-    - utter_anything_else
-* how_to_get_started{"product":"nlu"}
-    - slot{"product":"nlu"}
-    - utter_ask_for_nlu_specifics
-
-## Story from conversation with 201bb55841154f858f524a485f8816c3 on November 18th 2018
-
-* greet
-    - action_greet_user
-* how_to_get_started{"product":"nlu"}
-    - utter_getstarted
-    - utter_first_bot_with_rasa
-
-## Story from conversation with cd483ab3456d47dfb40bd1f51043fb54 on November 18th 2018
-
-* greet
-    - action_greet_user
-* greet
-    - action_greet_user
-* how_to_get_started{"product":"core"}
-    - utter_getstarted
-    - utter_first_bot_with_rasa
-
-## Story from conversation with cfa8bb9deaf0427498c662745431a282 on December 15th 2018
-* get_started_step1
-    - action_greet_user
-    - slot{"shown_privacy":true}
-* ask_whatisrasa
-    - action_chitchat
-* enter_data
-    - utter_not_sure
-    - utter_possibilities
-
-## Story from conversation with cdd14d763a664a5b95e998ce165bd86f on December 16th 2018
-* get_started_step1
-    - action_greet_user
-    - slot{"shown_privacy":true}
-* enter_data
-    - utter_not_sure
-    - utter_possibilities
-* how_to_get_started
-    - utter_getstarted
-    - utter_first_bot_with_rasa
-* affirm
-    - action_set_onboarding
-    - slot{"onboarding":true}
-    - utter_built_bot_before
-* deny
-    - utter_explain_stack
-    - utter_stack_details
-    - utter_explain_nlucore
-* affirm
-    - utter_explain_nlu
-    - utter_explain_core
-    - utter_direct_to_step2
-* enter_data
-    - utter_not_sure
-    - utter_possibilities
-
-## Story from conversation with 67a8696eb5894b25a800b6cbd7a695bb on December 15th 2018
-* get_started_step1
-    - action_greet_user
-    - slot{"shown_privacy":true}
-* ask_howdoing
-    - action_chitchat
-* ask_whoisit
-    - action_chitchat
-* enter_data
-    - utter_not_sure
-    - utter_possibilities
-
-## Story from conversation with 67a8696eb5894b25a800b6cbd7a695bb on December 15th 2018
-* get_started_step1
-    - action_greet_user
-    - slot{"shown_privacy":true}
-* ask_howdoing
-    - action_chitchat
-* ask_whoisit
-    - action_chitchat
-* enter_data
-    - utter_not_sure
-    - utter_possibilities
-* ask_faq_languages
-    - action_faqs
-* enter_data
-    - utter_not_sure
-    - utter_possibilities
```

Comment:
still confused as to why you've deleted this whole file

Reply:
It's been renamed to `interactive.md`.

Reply:
maybe i'm blind but i don't see that file anywhere, did it get lost in the PR port?

Reply:
My bad. I think it got lost there. I've added it now.